src/main/java/com/ea/restaurant/service/impl/Oauth2ServiceImpl.java

```diff
@@ -86,7 +86,6 @@ public Oauth2TokenResponseDto refreshToken(
       throws BadJOSEException, ParseException, JOSEException {
     validateClientCredentials(clientId, clientSecret);
     try {
-      Oauth2Util.validateToken(accessToken, oauth2SecretKey);
```

Comment:
why are you removing this?

Reply:
because the method getTokenDecode() bellow call validateToken().